azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -283,6 +289,28 @@ internal class CallCompositeActivity : AppCompatActivity() {
         }
     }
 
+    private fun actionBarAccessibilitySetup() {
+        try {
```

Comment:
Is try/catch on the entire block necessary?

I get there is a ton of places this can crash
(unchecked casts, .first() on finding the buttonView, etc).

It seems like it relies heavily on magic implementation details (i.e. is the AppCompatImageButton used, is it the first in the list of children, etc).

Reply:
Would moving to Toolbar help with a11y of this issue, I remember Mohtassim saying that it would for some of the other a11y issues like title/subtitle.

Reply:
@ahammer this code is hack to support accessibility focus. Currently we have only one button and if we need to add more, we need to setup custom action bar. 

Started a discussion to use custom action bar to support accessibility. -> Try catch is added to all as this piece of code is good if works, in any case we are not logging errors thus are only for debug purposes.